### ISSUE TITLE
Detect cluster version rather than requiring it as part of config

### ISSUE THREAD

**Issue by joeledwards** [npm-wharf/hikaru#5]
Having the right of a cluster is critical in order communicate. It is likely that this information is available as part of the cluster API and can be fetched via a simple GET request.

Figure out where this is defined in the API and perform an initial check against the cluster to determine its version before proceeding.

**Comment by joeledwards** [comment]
This information can be pulled from the `/version` route on the k8s manager (must be logged in).

```
λ aqui get -C -u <k8s-username> -p <k8s-password> https://<k8s-host>/version
{
  "major": "1",
  "minor": "8+",
  "gitVersion": "v1.8.7-gke.1",
  "gitCommit": "192ccad06d24af9828cbf42330e1d915cb586406",
  "gitTreeState": "clean",
  "buildDate": "2018-01-31T21:39:04Z",
  "goVersion": "go1.8.3b4",
  "compiler": "gc",
  "platform": "linux/amd64"
}

    connection : close
content-length : 269
  content-type : application/json
          date : Fri, 09 Feb 2018 20:21:30 GMT

GET https://<k8s-host>/version
```

**Comment by joeledwards** [comment]
This was implemented in version `1.13.0`
